### PR TITLE
JCSP-468: view backlog

### PR DESCRIPTION
**Pull request by ysemenkov**

### CODE REVIEW

File: custom-vu/src/main/kotlin/jces1209/vu/action/WorkOnSprint.kt [hunk 1]
```diff
@@ -5,94 +5,114 @@ import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
...
+    private val backlogsMemory: SeededMemory<ScrumBacklogPage>,
+    private val sprintsMemory: SeededMemory<ScrumSprintPage>
 ) : Action {
     private val logger: Logger = LogManager.getLogger(this::class.java)
 
     override fun run() {
-        val scrumBoard = scrumBoardsMemory.recall()
-        if (scrumBoard == null) {
+        workOnBacklog()
+
```
Comment (NataliiaShavshyna) [inline custom-vu/src/main/kotlin/jces1209/vu/action/WorkOnSprint.kt:23]: formatting

File: custom-vu/src/main/kotlin/jces1209/vu/action/WorkOnSprint.kt [hunk 2]
```diff
@@ -5,94 +5,114 @@ import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
...
+        workOnSprintPage()
+    }
+
+    private fun workOnSprintPage() {
+        val sprint = sprintsMemory.recall()
+        if (sprint == null) {
             logger.debug("I cannot recall scrum board, skipping...")
```
Comment (NataliiaShavshyna) [inline custom-vu/src/main/kotlin/jces1209/vu/action/WorkOnSprint.kt]: logger.debug("I cannot recall sprints, skipping...")

File: custom-vu/src/main/kotlin/jces1209/vu/action/WorkOnSprint.kt [hunk 3]
```diff
@@ -5,94 +5,114 @@ import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
...
         } else {
-            logger.debug("Scrum backlog doesn't contain enough issues to make reorder")
+            logger.debug("Scrum backlog doesn't contain sprint which is ready to be completed")
         }
+    }
+
+    private fun workOnBacklog(): Boolean {
+        val backlog = backlogsMemory.recall()
+        if (backlog == null) {
+            logger.debug("I cannot recall scrum board, skipping...")
```
Comment (NataliiaShavshyna) [inline custom-vu/src/main/kotlin/jces1209/vu/action/WorkOnSprint.kt]: logger.debug("I cannot recall backlog, skipping...")??

**Review by NataliiaShavshyna — CHANGES_REQUESTED** [review]

File: custom-vu/src/main/kotlin/jces1209/vu/action/WorkOnSprint.kt (hunk 1 above)
Comment (ysemenkov) [reply custom-vu/src/main/kotlin/jces1209/vu/action/WorkOnSprint.kt:23]: Do you mean the empty line? I like this one here - divider of different actions

**Review by NataliiaShavshyna — APPROVED** [review]

**Pull request merged**